<launch>
  <arg name="bayesopt_config" default="$(find bayesopt4ros)/configs/example_config_forrester.yaml"/>
  <node name="BayesOptServer" pkg="bayesopt4ros" type="bayesopt_server.py" args="-f $(arg bayesopt_config)" output="screen"/>
</launch>
